Feature/NewBookingFromTopMenu.feature

```diff
@@ -0,0 +1,31 @@
+Feature: New booking from the home page top menu
+
+  Scenario Outline: Making A New Booking from Home Page
+    Given The user is on the home page
+    When The user clicks on the 'BOOK NOW' top menu
+    Then User validates booking form is displayed
+
+    When The user clicks the input grid for guest information with value'<name>' and enters guest information '<guestName>'
```

Comment:
this is too long and unnecessary.
The user enters <guestName>'  in field 'guest name'
do not parameters the field name

Reply:
Done